src/libcore/slice/mod.rs

```diff
@@ -729,7 +729,8 @@ impl<T> [T] {
     /// Returns an iterator over `chunk_size` elements of the slice at a
     /// time. The chunks are slices and do not overlap. If `chunk_size` does
     /// not divide the length of the slice, then the last up to `chunk_size-1`
-    /// elements will be omitted.
+    /// elements will be omitted and can be retrieved from the `remainder`
```

Comment:
This and the same for `exact_chunks_mut` can't be linked to the documentation of the function. It would have to be `../std/slice/etc` for the direct documentation here and `../../std/slice/etc` for the "copied" documentation in e.g. `Vec` (for its `AsRef<_>` impl)

Just omitting the link for now

Reply:
https://github.com/rust-lang/rust/issues/32130 Is the relevant issue for this AFAIU